libmscore/chord.cpp

```diff
@@ -2897,7 +2897,7 @@ QPointF Chord::layoutArticulation(Articulation* a)
 
 void Chord::reset()
       {
-      score()->undoChangeProperty(this, P_ID::STEM_DIRECTION, Direction(Direction::AUTO));
+      score()->undoChangeProperty(this, P_ID::STEM_DIRECTION, Direction::AUTO);
```

Comment:
oops, https://github.com/musescore/MuseScore/commit/8aff920cfd84c8ead4834874da3b0fb2732fc3d7 is wrong. It must be read Direction(Direction::AUTO), sorry for the confusion.
It seems to be an example of why automatic type conversions are bad.